RELEASENOTES.md

```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+### 3.6.2 (January 9, 2019)
```

Comment:
please remove the date and put `UNRELEASED`, the release date is put right before the release happens.

Reply:
Done